src/Common/MapGenerator.ts

```diff
@@ -571,7 +571,7 @@ export class MapGenerator
     
                             this._log("---------- " + p.documentation.raw)
 
-                            options[p.name.raw] = bodyPlaceholder;
+                            options[m.name.raw + p.name.raw] = bodyPlaceholder;
```

Comment:
how does it look when concatenated? could you give and example?

Reply:
actually when you don't do that, it will some how override the previous parameters. as you can see from az issue 66, the screencut code is actually the "parameters"' definition of "patch" operation. and if you move "patch" operation in front of "createorupdate" operation, the "createorupdate"'s "parameters"  will override the "patch"'s "parameters".
I finally find the problem is in this MapGenerator file.